### PR TITLE
HTTP route discovery

### PR DESCRIPTION
**Pull request by jessemillar**
Copying work from PR 12 to implement route discovery as mentioned in https://github.com/goshlanguage/cerulean/issues/15

### CODE REVIEW

**Comment by jessemillar** [comment]
As alluded to in [this comment](https://github.com/goshlanguage/cerulean/issues/15#issuecomment-704537131), I'm still investigating a slightly cleaner/user friendly way of adding and routing a new service (since we're going to be doing that a lot during the development of this library). If the current code ends up being the best we can do, I'll add some docs in the form of `README.md` file(s) that explain how to add a new service to help new contributors not be confused.

**Comment by jessemillar** [comment]
Did more Google perusing and couldn't find a satisfactory method for discovering routes at runtime. Added docs to this PR for now. Open to discussion.

File: cerulean.go [hunk 1]
```diff
@@ -28,6 +29,7 @@ type Cerulean struct {
 //   in order to point it at the mock server.
 func New() Cerulean {
 	e := echo.New()
+	e.HideBanner = true // Make log output less noisy by removing ASCII artwork
```
Comment (goshlanguage) [inline cerulean.go:32]: :100:

**Comment by goshlanguage** [comment]
Great stuff!

**Pull request merged**